README.md

```diff
@@ -194,6 +194,16 @@ of more than one.
 
 `bundle exec rake`
 
+### Testing in a browser
+
+If you want to test this behaviour in a browser without Fastly set up (in dev
```

Comment:
The GOV.UK Chrome extension should do this in all environments, so this shouldn't be necessary?